lib/util/cache/repository/impl/cache-client-factory.ts

```diff
@@ -0,0 +1,37 @@
+import { GlobalConfig } from '../../../../config/global';
+import type { RepositoryCacheType } from '../../../../config/types';
+import { logger } from '../../../../logger';
+import type { CacheClient } from '../types';
+import { LocalRepoCache } from './client/local';
+
+export class CacheClientFactory {
```

Comment:
```suggestion
export class CacheFactory {
```

remove `client` from all other too